.docs/Get-VSTeamProcess.md

```diff
@@ -29,18 +27,32 @@ This will return all the Process Templates
 ### -------------------------- EXAMPLE 2 --------------------------
 
 ```PowerShell
-PS C:\> Get-VSTeamProcess -top 5 | Format-Wide
+PS C:\> Get-VSTeamProcess  | Format-Wide
```

Comment:
Why get rid of the -Top 5 example? Just piping to FW is not teaching how to use Get-VSTeamProcess.

Reply:
Because the API ignores top. 
When I changed the source for processes (see below for the explanation) I found that -top 5 returned all the processes in a live project.  I also found that $skip=0 never got into the query URI. I know now it is done on purpose ... 
Overnight I had an idea. _If the user specifies top or skip call the process/processes API not the the work/processes API_  Tested that, and I got all the processes. It's not the function, and it's nothing to do with whether skip is there or not. A URI with $top=5 and/or $skip=2 returns everything. 

I can't find the docs page I was looking at this morning which showed top and skip but this one (which I swear is in another part of the tree)  does not mention them 
https://docs.microsoft.com/en-us/rest/api/azure/devops/core/processes/get?view=azure-devops-rest-5.1

So.... the -top and -skip and parameters are now ignored. If someone is using them I don't want their code to break so the parameters are still there but hidden, and they have been expunged from the help .